docs/src/inverse_problems/petab_ode_param_fitting.md

```diff
@@ -0,0 +1,494 @@
+
+### Creating a PEtabModel
+Finally, we combine all inputs in a single `PEtabModel`. To it, we also pass the initial conditions of our simulations (using the `state_map` argument). It is also possible to have [initial conditions with uncertainty](@ref petab_simulation_initial_conditions_uncertainty), [that vary between different simulations](@ref petab_simulation_conditions), or [that we attempt to fit to the data](@ref petab_simulation_initial_conditions_fitted).
+```@example petab1
+petab_model = PEtabModel(rn, observables, measurements, params; state_map=u0)
+nothing # hide
+```
+
+### Fitting parameters
+We are now able to fit our model to the data. First, we create a `PEtabODEProblem`. Here, we use `petab_model` as the only input, but it is also possible to set various [numeric solver and automatic differentiation options](@ref petab_simulation_options) (such as method or tolerance).
+```@example petab1
+petab_problem = PEtabODEProblem(petab_model)
+nothing # hide
+```
+Since no additional input is given, default optiosn are selected by PEtab.jl (and generally, its choices are good).
+
+To fit a parameter set we use the `calibrate_model` function. In addition to our `PEtabODEProblem`, we must also provide an initial guess (which can be generated with the `generate_startguesses` function) and an optimisation algorithm (which needs to be imported specifically). PEtab.jl supports various [optimisation methods and options](@ref petab_optimisation_optimisers). 
+```@example petab1
+using Optim
+p0 = generate_startguesses(petab_problem, 1)
+res = calibrate_model(petab_problem, p0, IPNewton())
+```
+
+We can now simulate our model for the fitted parameter set, and compare the result to the measurements and true solution.
+```@example petab1
+oprob = ODEProblem(rn, u0, (0.0, 10.0), get_ps(res, petab_problem))
+sol = solve(oprob, Tsit5(); saveat=0.1)
+plot!(sol; idxs=4, label="Fitted solution")
+```
+Here we use the `get_ps` function to retrieve a full parameter set using the optimal parameters. Alternatively, the `ODEProblem` or fitted simulation can be retrieved directly using the `get_odeproblem` or `get_odesol` [functions](https://sebapersson.github.io/PEtab.jl/dev/API_choosen/#PEtab.get_odeproblem), respectively (and the initial condition using the `get_u0` function). The calibration result can also be found in `res.xmin`, however, note that PEtab automatically ([unless a linear scale is selected](@ref petab_parameters_scales)) converts parameters to logarithmic scale, so typically `10 .^res.xmin` are the values of interest. If you investigate the result from this example you might note that the found parameter set (while it fits the data well) does not correspond to the true parameter set. This phenomenon is related to *identifiability*, and is very important for parameter fitting.
```

Comment:
There is an upcoming documentation on identifiability. The idea is to (when it is done, which should be very soon) have this link that.

Reply:
OK, it still seems like a poor advertisement for PETab to pick an example it can't actually recover the initial parameters on (even if it is because the example is not identifiable). 

I'd still suggest dropping the non-identifiable example as the opening example where users are getting the flavor for using PETab. It will be more convincing that it is useful if people see it can do a decent job recovering the parameters (and I would show the fit vs. exact values to make this clear).

Reply:
You can then have a non-identifiable example as a second example. (Though I would instead suggest that the opening inverse problem tutorial should instead cover such things since it is more of a generic fitting problem.)

Reply:
I have reworded it to say
"If you investigate the result from this example you might note, that even if PEtab.jl has found the global optimum (which fits the data well), this does not actually correspond to the true parameter set. This phenomenon is related to the concept of *identifiability*, which is very important for parameter fitting."
If people are happy with this we will go ahead and merge this.